### PR TITLE
Add convert.com experimentation sandbox (Fixes #7508)

### LINKED ISSUE #7508: Build a sandbox for CRO
## Description

Implement the sandbox defined in #7505: https://docs.google.com/document/d/1RXzBVUk3LSbyBeJOesPrKcekvA5bOUBiUIiQy7frHgQ/edit?usp=sharing

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Sandbox is built and a) ready to port content to b) ready to send audience to

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds a new `/exp/` app to bedrock for experiment pages.
- Implements a new base template for the `/exp/` app that includes [Convert](https://www.convert.com/).
- Adds an opt-out page at `/exp/opt-out/`
- Adds two simplified, experimental landing page clones:
  - `/en-US/exp/firefox/new/`
  - `/de/exp/firefox/new/` (hard-coded strings)
- Updates websites privacy notice.

## Issue / Bugzilla link
#7508

## Testing
Demo: https://www-demo3.allizom.org/en-US/exp/firefox/new/

- [x] Experiment pages should should display correctly.
- [ ] Experiment pages should have a canonical URL pointing to their non-experimental equivalents.
- [x] Convert script should not load when DNT is enabled.
- [ ] Opt-out page should work as expected.
- [ ] Websites privacy notice should be updated to reference Convert.

### CODE REVIEW

File: bedrock/settings/base.py [hunk 1]
```diff
@@ -1407,10 +1411,15 @@ def get_app_name(hostname):
     'tagmanager.google.com',
     'www.youtube.com',
     's.ytimg.com',
+    'cdn-3.convertexperiments.com',
+    'app.convert.com',
+    '1003350.track.convertexperiments.com',
+    '1003343.track.convertexperiments.com',
```
Comment (amychurchwell) [inline bedrock/settings/base.py]:
```
    '1003350.track.convertexperiments.com',
    '1003343.track.convertexperiments.com',
```
I saw that these numbers correlate to our project directory IDs in convert. Is this secure? Do those ID numbers need to be secret?

**Comment by amychurchwell** [comment]
@alexgibson I may need your guidance reviewing this PR.

File: bedrock/settings/base.py (hunk 1 above)
Comment (alexgibson) [reply bedrock/settings/base.py]: There's nothing in this PR that needs to remain secret. This is all visible information that's available to anyone who opens dev tools, much like our Google Analytics ID.

**Comment by alexgibson** [comment]
@amychurchwell I sent a zoom invite so we can talk through this PR 👍

**Comment by amychurchwell** [comment]
@hoosteeno should we have this copy reviewed by legal? https://www-demo3.allizom.org/en-US/exp/opt-out/
I know that different countries have different requirements/laws in regards to data privacy.

**Comment by alexgibson** [comment]
Just noticed another CSP violation on Sentry. I’ll need to add an exception for this:

```
CSP Violation: https://data.track.convertexperiments.com/getjs/global/data.js?client_id=10039&project_id=1003350&do=preview&exp=10031201&var=10033400&_rnd=1568828433794
```

**Comment by hoosteeno** [comment]
Legal has signed off on the opt-out page -- no changes needed.

**Comment by alexgibson** [comment]
Thanks @amychurchwell @hoosteeno, merging!

**Pull request merged**